### DISCUSSION TITLE
article/notion-next-comment-plugin

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#49] [Announcements]
# article/notion-next-comment-plugin

NotionNext添加Cusdis/Giscus/Gitalk/Utterance的步骤教程

https://tangly1024.com/article/notion-next-comment-plugin

**Comment by fixcode-h** [comment]
1111

**Comment by tangly1024** [comment]
因评论区访问量过大，暂关闭其他评论渠道，只保留giscus

**Comment by Mozilan1216** [comment]
你好
